src/lib/client.ts

```diff
@@ -59,6 +69,9 @@ export class Permanent {
       const archive = await this.session.getAccountArchive();
       // get the default archiveNbr from the account
       this.archiveNbr = archive.archiveNbr;
+      if (archive.archiveId !== undefined) {
+        this.archiveId = archive.archiveId;
```

Comment:
When is `archive.archiveId` undefined? Does the server sometimes not include the `archiveId` field in the response? Or was this intended to be `if (this.archiveId === undefined)`?

Reply:
I don't remember exactly what made me write this, but I presume it was in reaction to an error I ran into.  I can't think of a time when the server would return an `archiveNbr` and not an `archiveId`, though.

Reply:
Removed this.